setup/src/include/central.cpp

```diff
@@ -20,6 +20,8 @@ struct catagory CATS;
 struct group GROUPS;
 struct global_settings CONFIG;
 
+static char NULL_CHAR = '\0';
```

Comment:
This is a weird way of passing an empty string.

Why not

``` C++
static char EMPTY_STR[] = "";
```

Reply:
Good point. I'll fix that.